code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1559,6 +1559,53 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "cup of sake"
 	glass_desc = "A traditional cup of sake."
 
+/datum/reagent/consumable/ethanol/peppermint_patty
+	name = "Peppermint Patty"
+	id = "peppermint_patty"
+	description = "This lightly alcoholic drink combines the benefits of menthol and cocoa."
+	color = "#45ca7a"
+	taste_description = "mint and chocolate"
+	boozepwr = 25
+	glass_icon_state = "peppermint_patty"
+	glass_name = "Peppermint Patty"
+	glass_desc = "A boozy minty hot cocoa that warms your belly on a cold night."
+
+/datum/reagent/consumable/ethanol/peppermint_patty/on_mob_life(mob/living/M)
+	M.adjust_bodytemperature(5 * TEMPERATURE_DAMAGE_COEFFICIENT, 0, BODYTEMP_NORMAL)
+	..()
+
+/datum/reagent/consumable/ethanol/candyland_extract
+	name = "Candyland Extract"
+	id = "candyland_extract"
+	description = "This mesmerizing drink has it's properties from the mercury and space drugs. consumption not suggested."
+	color = "#664300"
+	taste_description = "the rainbow" //taste the rainbow!
+	boozepwr = 0
+	glass_icon_state = "candyland_extract"
+	glass_name = "Candyland Extract"
+	glass_desc = "A very nice shifting rainbow in a glass!"
+
+/datum/reagent/consumable/ethanol/candyland_extract/on_mob_add(mob/living/M)
+	for(var/turf/globalturf in world) //disaster strikes
```

Comment:
so i know this aint right


but how is it wrong :)

Reply:
Looks like you've picked up more than just S____'s ideas.

Reply:
💃 acdc - var/turf/T all around the world!

no but really why can't i get away with this

Reply:
Have you tested it?

Reply:
You may as well just make a new layer for turfs and add the candy overlay to them on initialize then just add and remove the ability to see them when ingested.

Reply:
but wouldn't that incur the ire of maintainers

Reply:
\>`on_mob_add`
\>`in world` loop
dear god

Reply:
shoulda thought harder about it sorry